LICENSE.txt

```diff
@@ -849,9 +849,9 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 --------------------------------------------------------------------------------
 
-The files in cpp/src/arrow/vendored/utf8cpp/ have the following license
+The files in cpp/src/arrow/vendored/utfcpp/ have the following license
 
-Copyright 2006 Nemanja Trifunovic
+Copyright 2006-2016 Nemanja Trifunovic
```

Comment:
2018? There is 2018 at https://github.com/apache/arrow/pull/8234/files#diff-152b5a1d1b78a0d30f038832fc83bfbcR1 .

```suggestion
Copyright 2006-2018 Nemanja Trifunovic
```